### PR TITLE
JBoss EAP and Wildfly

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+
+### Running DXP on JBoss EAP in Standalone Mode vs. Domain Mode
+
+JBoss EAP can be launched in either *standalone* mode or *domain* mode. Domain mode allows multiple application server instances to be managed from a single control point. A collection of such application servers is known as a *domain*. For more information on standalone mode vs. domain mode, please refer to the section on this topic in the [JBoss EAP Product Documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html/introduction_to_jboss_eap/overview_of_jboss_eap#operating_modes).
+
+DXP supports JBoss EAP when it runs in standalone mode but not when it runs in domain mode. DXP's hot-deploy does not work with a managed deployment, since JBoss manages the content of a managed deployment by copying files (exploded or non-exploded). This prevents JSP hooks and Ext plugins from working as intended. For example, JSP hooks don't work on JBoss EAP running in managed domain mode, since DXP's JSP override mechanism relies on the application server. Since JSP hooks and Ext plugins are deprecated, however, you may not be using them.
```
Comment (jhinkey) [inline docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md:76]: @justinchoi001 You should replace "hot-deploy" with "auto deploy" as DXP doesn't have hot-deploy. Auto deploy is DXP's mechanism for copying files from the deploy/ folder to the appropriate osgi/ folder that our OSGi container uses for that file.

Comment (justinchoi001) [reply docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md:76]: Thanks Jim! I've updated the PR

**Comment by sez11a** [comment]
Just started reviewing :)

**Comment by sez11a** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/395

**Pull request closed without merging**